wrfhydropy/core/domain.py

```diff
@@ -164,7 +164,7 @@ def copy_files(self,dest_dir: str,symlink: bool=True):
         # Symlink in nudging files
 
         # handling nudging obs files
-        if self.nudging_dir is not None:
+        if not (self.nudging_dir is None or self.nudging_dir is ''):
```

Comment:
hmm this seems odd, why would it be an empty string?

Reply:
If someone passes an empty string...

Reply:
documented in the code.